### ISSUE TITLE
usage within a docker container

### ISSUE THREAD

**Issue by jmls** [artoale/brackets-terminal#37]
I'm trying to get brackets running as a docker container - so far so good, the basics are there, thanks to https://github.com/rabchev/brackets-server . 

However, I can't get brackets-terminal working. I can install it, but it just doesn't start .

Is there any debug that I can supply that would help sort this ?

thanks!

**Comment by jmls** [comment]
a month on .. is this project dead ? Are there any alternatives ?

The problem is this : localhost is hardcoded into the connection. This obviously does not work if the host is not localhost ;)

**Comment by Jeff-Russ** [comment]
@jmls  are you using Mac OS X because I may have a solution.

**Comment by jmls** [comment]
nope, not using OS X - but I found the problem ;) Turns out that brackets-server requires either TMP or TEMP env variables to be set (I haven't figured out which one yet)

thanks anyway
